dockerfiles/vcpkg/centos-7.dockerfile

```diff
@@ -1,9 +1,21 @@
 ARG VCPKG_REF=latest
-FROM hpccbuilds/vcpkg-centos-7:$VCPKG_REF
+ARG DOCKER_NAMESPACE=hpccbuilds
+FROM ${DOCKER_NAMESPACE}/vcpkg-centos-7:$VCPKG_REF
```

Comment:
This will change shortly (you may need to change it depending on which PR get pulled first)